code/src/java/pcgen/gui2/dialog/DataInstaller.java

```diff
@@ -180,6 +274,134 @@ else if (source == installButton)
 				}
 			}
 		}
+
+
+		/**
+		 * Read data set.
+		 *
+		 * @param dataSet the data set
+		 *
+		 * @return true, if successful
+		 */
+		private boolean readDataSet(File dataSet)
+		{
+			// Open the ZIP file
+			try (ZipFile in = new ZipFile(dataSet)) {
+				// Get the install file in a case insensitive manner
+				ZipEntry installEntry = null;
+				@SuppressWarnings("rawtypes")
+				Enumeration entries = in.entries();
+				while (entries.hasMoreElements()) {
```

Comment:
I think you need to grab our formatting rules and apply them in your editor. Braces such as these should be on their own line.